src/v3/plugins/github/nodes.js

```diff
@@ -99,82 +122,82 @@ export function fromRaw(x: RawAddress): StructuredAddress {
   }
   const [_unused_sc, _unused_gh, kind, ...rest] = NodeAddress.toParts(x);
   switch (kind) {
-    case "REPO": {
+    case REPO_TYPE: {
       if (rest.length !== 2) {
         throw fail();
       }
       const [owner, name] = rest;
-      return {type: "REPO", owner, name};
+      return ({type: REPO_TYPE, owner, name}: RepoAddress);
     }
-    case "ISSUE": {
+    case ISSUE_TYPE: {
       if (rest.length !== 3) {
         throw fail();
       }
       const [owner, name, number] = rest;
-      const repo = {type: "REPO", owner, name};
-      return {type: "ISSUE", repo, number};
+      const repo = {type: REPO_TYPE, owner, name};
+      return ({type: ISSUE_TYPE, repo, number}: IssueAddress);
     }
-    case "PULL": {
+    case PULL_TYPE: {
       if (rest.length !== 3) {
         throw fail();
       }
       const [owner, name, number] = rest;
-      const repo = {type: "REPO", owner, name};
-      return {type: "PULL", repo, number};
+      const repo = {type: REPO_TYPE, owner, name};
+      return ({type: PULL_TYPE, repo, number}: PullAddress);
     }
-    case "REVIEW": {
+    case REVIEW_TYPE: {
       if (rest.length !== 4) {
         throw fail();
       }
       const [owner, name, pullNumber, id] = rest;
-      const repo = {type: "REPO", owner, name};
-      const pull = {type: "PULL", repo, number: pullNumber};
-      return {type: "REVIEW", pull, id};
+      const repo = {type: REPO_TYPE, owner, name};
+      const pull = {type: PULL_TYPE, repo, number: pullNumber};
+      return {type: REVIEW_TYPE, pull, id};
```

Comment:
Type assertions on others, but not here. Make consistent?

Reply:
Done